spec/services/teachers/induction_spec.rb

```diff
@@ -2,91 +2,87 @@
   subject(:service) { described_class.new(teacher) }
   let(:teacher) { FactoryBot.create(:teacher) }
 
-  describe "#current_induction_period" do
-    before do
-      FactoryBot.create(:induction_period, teacher:, started_on: 2.years.ago, finished_on: 1.year.ago)
-    end
+  let(:current_period) do
+    FactoryBot.create(:induction_period, :active, teacher:, started_on: 6.months.ago)
+  end
 
-    it "returns the induction period without a finished_on date" do
-      current_period = FactoryBot.create(:induction_period, :active, teacher:, started_on: 6.months.ago)
+  let(:recent_finish) do
```

Comment:
Recent is subjective, I had to scroll back up to see the definition, how do you feel about being more explicit here, something like:

`let(:induction_period_finished_one_year_ago)`

This will also help to reduce cognitive load, especially for someone new to the codebase.